tiling.js

```diff
@@ -2839,13 +2845,31 @@ function grabEnd(metaWindow, type) {
     inGrab = false;
 }
 
+function setAllWorkspacesInactive() {
+    try {
+        for (let i = 0; i < workspaceManager.get_n_workspaces(); i++) {
+            const ws = workspaceManager.get_workspace_by_index(i);
```

Comment:
maybe change this to `let` (moreso  convention, although `const` works here in this use case `let` is more appropriate).